jetty-osgi-servlet-api/pom.xml

```diff
@@ -135,12 +135,26 @@
                <transformers>
                 <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
                   <resource>META-INF/MANIFEST.MF</resource>
-                  <file>target/classes/META-INF/MANIFEST.MF</file>
+                  <file>${project.build.outputDirectory}/META-INF/MANIFEST.MF</file>
                 </transformer>
+                 <transformer implementation="org.eclipse.jetty.toolchain.shade.MergeFilesTransformer">
+                   <resource>META-INF/LICENSE</resource>
+                   <files>
+                     <file>${project.build.directory}/sources/META-INF/LICENSE</file>
+                     <file>${project.build.directory}/sources/META-INF/LICENSE.txt</file>
```

Comment:
Oddly named files.
Is one Eclipse License? the other CDDL? (or something)

Reply:
@joakime yes...